### ISSUE TITLE
Find a solution to Matrix->Matrix message relay in multimatrix bridged rooms

### ISSUE THREAD

**Issue by leonerd** [matrix-org/matrix-appservice-slack#27]
Currently there's no neat handling of the case of a BridgedRoom with multiple matrix rooms attached; see d7052fb.

**Comment by leonerd** [comment]
On further discussion, it seems like this would be best handled by having a more general way to work with incoming bot messages; i.e. anything that doesn't come from a regular slack user. People will want the slack bridge to handle incoming bot messages from other things anyway, so that mechanism can do matrix->matrix (internal) reflections

**Comment by leonerd** [comment]
Multimatrix rooms aren't implemented any more in latest webhook-routing code, so this can be deferred somewhat. Also depends on #29

**Comment by ara4n** [comment]
Just to make it clear: we *need* multimatrix rooms, and we should just 
bridge the traffic via the remote network.
